src/pages/Chat/steps/InsuranceTypeInput.test.tsx

```diff
@@ -24,6 +24,11 @@ it('handles form changes', () => {
     </Provider>,
   )
 
+  wrapper
+    .find('input#size')
+    .simulate('change', { target: { value: 'characters' } })
```

Comment:
maybe we could add some numbers here just to be sure it doesn't swallow everything?

Reply:
so something like `simulate 1blargh2blargh2` -> `expect 123`?